app/(protected)/quiz/[id]/error.tsx

```
@@ -0,0 +1,17 @@
+"use client";
+
+type QuizErrorProps = {
+  error: Error;
+  reset: () => void;
+};
+
+export default function Error({ error, reset }: QuizErrorProps) {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)